packages/gestalt/src/Spinner/VRSpinner.tsx

```diff
@@ -6,40 +7,88 @@ import { useDefaultLabelContext } from '../contexts/DefaultLabelProvider';
 const SIZE_NAME_TO_PIXEL = {
   sm: 32,
   md: 40,
-  lg: 48,
+  lg: 56,
 } as const;
 
-type Props = {
-  accessibilityLabel?: string;
-  delay?: boolean;
+type SpinnerBodyProps = {
+  accessibilityLabel: string;
+  delay: boolean;
   show: boolean;
-  size?: 'sm' | 'md' | 'lg';
+  size: 'sm' | 'md' | 'lg';
+  onExitAnimationEnd: () => void;
```

Comment:
https://gestalt.pinterest.systems/web/overlaypanel#Props-onAnimationEnd

Can we keep similar API as in sidenavigation?

onAnimationEnd | (arg1: { animationState: "out" }) => void
-- | --




onAnimationEnd | (arg1: { animationState: "in" \| "out" }) => void
-- | --